### PR TITLE
tracker: Add `-Wno-int-conversion` to fix FTBFS on recent compilers

### PR DESCRIPTION
**Pull request by qwertychouskie**
###### Type(s)

- [x] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 14.4.1 23E224 arm64
Xcode 15.3 15E204a

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?
- [x] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: gnome/tracker/Portfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ PortGroup           gobject_introspection 1.0
 
 name                tracker
 version             1.12.4
-revision            15
+revision            16
```
Comment (reneeotten) [inline gnome/tracker/Portfile:9]: is this only fixing a build failure; if so, you should not increase the "revision".

File: gnome/tracker/Portfile [hunk 2]
```diff
@@ -123,6 +123,8 @@ configure.args      --disable-schemas-compile \
                     --enable-icon \
                     --enable-playlist
 
+configure.cflags-append -Wno-int-conversion
```
Comment (reneeotten) [inline gnome/tracker/Portfile:126]: is there an upstream issue that tracks the problem and/or do they have a solution lined up already (which presumably will be different than this)?

File: gnome/tracker/Portfile (hunk 1 above)
Comment (qwertychouskie) [reply gnome/tracker/Portfile:9]: I'll change it back then

File: gnome/tracker/Portfile (hunk 2 above)
Comment (qwertychouskie) [reply gnome/tracker/Portfile:126]: AFAIU, Tracker 1.x has not been supported in many, many years.  When I get the chance, I want to try to update to 2.x so I can update Nautilus to a semi-modern version, but in the mean time, this is a simple stop-gap.

**Comment by qwertychouskie** [comment]
Looks like everything passes after the rebase, can this be merged now?

**Pull request merged**